src/types/Posts.ts

```diff
@@ -0,0 +1,10 @@
+import { Comments } from './Comments';
+import { Users } from './Users';
+
+export interface Posts {
```

Comment:
```suggestion
export interface Post {
```

You describe one entity here